site/kube-addon.md

```diff
@@ -77,10 +77,21 @@ definition](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitio
 
 ###<a name="blocked-connections"></a> Troubleshooting Blocked Connections
 
-If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, check the `weave-npc` container's logs:
+If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, check the `weave-npc` container's logs.
+
+To do this, first you have to find the name of the Weave Net pod running on the relevant host:
+
+```
+$ kubectl get pods -n kube-system -o wide | grep weave-net
+weave-net-08y45                  2/2       Running   0          1m        10.128.0.2   host1
+weave-net-2zuhy                  2/2       Running   0          1m        10.128.0.4   host3
+weave-net-oai50                  2/2       Running   0          1m        10.128.0.3   host2
+```
+
+Pick the one you need - e.g. if you want to look at host2 then pick `weave-net-oai50`, and do:
 
```

Comment:
Select the relevant container, for example, if you want to look at host2 then pick `weave-net-oai50` and run: